# Currently available actions are:
# request_config
# update_config
string action

# Temoto namespace where the configuration belongs to.
string temoto_namespace

# byte array that contains the serialized payload
uint8[] payload
